amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,93 @@
+= Troubleshooting Amazon SQS Connector 5.11 - Mule 4
+
+To troubleshoot Anypoint Connector for Amazon SQS Connector, familiarize yourself with the information about interpreting commonly thrown exception messages. 
+
+== Understand Commonly Thrown Exceptions
+
+Here is a list of commonly thrown exception messages and how to interpret them:
+
+* SQS:ACCESS_DENIED
+
+ The user does not have permission to execute this operation. 
+
+* SQS:CONNECTIVITY
+
+ The connection to the Amazon SQS server is interrupted.
+
+* SQS:INTERNAL_FAILURE
+    
+ The request is failing due to an unknown error, exception, or failure.
+
+* SQS:INVALID_CREDENTIALS
+
+ Authentication against the SFTP server is failing due to invalid credentials. 
+
+* SQS:INVALID_DATA
+
+ The test connection cannot be performed because the test queue arn parameter is not set.
+
+* SQS:MESSAGE_NOT_IN_FLIGHT
+
+ The specified queue recieved a PurgeQueue request within the last 60 seconds. 
+
+* SQS:MESSAGE_SIZE_THRESHOLD_OUT_OF_RANGE
+
+ Validation error.
+
+* SQS:OPT_IN_REQUIRED
+
+ The AWS access key ID needs a subsription for the service.
+
+* SQS:OVER_LIMIT
+
+ The attempted action exceeds a limit. 
+ This happens, for example, if RecieveMessages recieves the maximum number of inflight messages or if the queue reaches the maximum number of permissions for ADDPermission. 
+
+* SQS:PARAMETER_OUT_RANGE
+
+ Validation error thrown when the server returns AmazonS3Exception with 403 HTTP status code.
+
+* SQS:PURGE_QUEUE_IN_PROGRESS
+
+ The specified queue recieved a PurgeQueue request within the last 60 seconds. 
+
+* SQS:QUEUE_ALREADY_EXISTS
+
+ There is a queue with this name already and the request includes different attributes from the existing queue with this name.
+
+* SQS:QUEUE_DELETED_RECENTLY
+
+ A queue with the same name was deleted within the last 60 seconds. 
+
+* SQS:REQUEST_EXPERIED
+
+ The request reaches the service more than 15 minutes after the date stamp on the request, more than 15 minutes after the request expiration date, or the date stamp on the request is more than 15 minutes in the future.
+
+* SQS:RETRY_EXHAUSTED
+
+ Indicates that retries of a certain execution block are exhausted. 
+
+* SQS:SERVICE_UNAVAILABLE
```

Comment:
You may have to move this SQS:SERVICE_UNAVAILABLE throw after SQS:S3_BUCKET_NOT_FOUND for alphabetical order